src/odemis/driver/test/xt_client_test.py

```diff
@@ -614,6 +615,20 @@ def setUp(self):
             self.skipTest("Chamber needs to be in vacuum, please pump.")
         self.xt_type = "xttoolkit" if "xttoolkit" in self.microscope.swVersion.lower() else "xtlib"
 
+    def test_get_value_from_software_version(self):
```

Comment:
can you add a test for `transfer_latest_package` and maybe by default skip it, so we do not transfer a package every time we run these test cases

Reply:
Sure, will do so

Reply:
I added an additional check [here](https://github.com/nandishjpatel/odemis/blob/46cd9bf7c74b01ffd64e4645496909b7a3fbaba7/src/odemis/driver/xt_client.py#L256-L258)

Reply:
is that the correct link, I don't see it there

Reply:
I believe [this additional](https://github.com/nandishjpatel/odemis/blob/5b35cbdf82b67cc7527465255a36a921d90324ac/src/odemis/driver/xt_client.py#L335) check if it's a proper zip file overcomes the need for [transfer_latest_package](https://bitbucket.org/delmic/xtlib-adapter/src/9708c2ef2f5b15cee2fd9a0ff0688c261a4dfbc0/xtadapter/base_adapter.py#lines-1438) test-case because standard modules are being used. What do you suggest @tepals?